### PR TITLE
Add supported method to get leverage

### PR DESCRIPTION
**Pull request by lamtrhieu**
The purpose of the pull request :
- Change the response of method create/cancel for margin orders. Don't map to struct, but return raw data from api
- Remove lots of duplicate code in margin order/account and futures order/account
- Add new method to get account leverage.

### CODE REVIEW

**Review by kilu01 — APPROVED** [review]
lgtm

File: lib/ex_huobi/margin/rest/account.ex [hunk 1]
```diff
@@ -15,33 +16,10 @@ defmodule ExHuobi.Margin.Rest.Account do
...
-        {:error, error}
-    end
-  end
-
-  defp parse_to_obj(data) when is_list(data) do
-    data
-    |> Enum.map(&to_struct/1)
+    HTTPClient.get(@margin_endpoint, "/v1/account/accounts", config)
+    |> Handler.parse_response()
+    |> Util.transform_response_data(AccountModel)
```
Comment (sirius248) [inline lib/ex_huobi/margin/rest/account.ex:22]: If do piping, let pipe all the way
```elixir
@margin_endpoint
|> HTTPClient.get("/v1/account/accounts", config)
|> Handler.parse_response()
|> Util.transform_response_data(AccountModel)
```

File: lib/ex_huobi/margin/rest/order.ex [hunk 2]
```diff
@@ -17,14 +18,22 @@ defmodule ExHuobi.Margin.Rest.Order do
 
   @spec get(order_id, config) :: response
   def get(order_id, config) do
-    case HTTPClient.get(@margin_endpoint, "/v1/order/orders/#{order_id}", config)
-         |> Handler.parse_response() do
-      {:ok, data} ->
-        {:ok, data |> parse_to_obj()}
+    HTTPClient.get(@margin_endpoint, "/v1/order/orders/#{order_id}", config)
```
Comment (sirius248) [inline lib/ex_huobi/margin/rest/order.ex]: Similar here too

File: lib/ex_huobi/margin/rest/order.ex [hunk 3]
```diff
@@ -17,14 +18,22 @@ defmodule ExHuobi.Margin.Rest.Order do
...
+  ## Examples
 
-      {:error, _} = error ->
-        error
-    end
+    iex> ExHuobi.Margin.Rest.Order.get_open(%{"account-id": 12035991, symbol: "btcusdt"}, config)
+  """
+  @spec get_open(params, config) :: response
+  def get_open(params, config) do
+    HTTPClient.get(@margin_endpoint, "/v1/order/openOrders", params, config)
```
Comment (sirius248) [inline lib/ex_huobi/margin/rest/order.ex]: Same

File: lib/ex_huobi/margin/rest/order.ex [hunk 4]
```diff
@@ -44,14 +53,8 @@ defmodule ExHuobi.Margin.Rest.Order do
...
   def create(params, config) do
-    case HTTPClient.post(@margin_endpoint, "/v1/order/orders/place", params, config)
-         |> Handler.parse_response() do
-      {:ok, data} ->
-        {:ok, %Order{id: data}}
-
-      {:error, _} = error ->
-        error
-    end
+    HTTPClient.post(@margin_endpoint, "/v1/order/orders/place", params, config)
```
Comment (sirius248) [inline lib/ex_huobi/margin/rest/order.ex]: Here too

File: test/futures/rest/account_test.exs [hunk 5]
```diff
@@ -3,20 +3,50 @@ defmodule ExHuobi.Futures.Rest.Account.Test do
...
   use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
-  @config %ExHuobi.Config{
-    api_key: System.get_env("HUOBI_API_KEY"),
-    api_secret: System.get_env("HUOBI_API_SECRET")
-  }
+  # @config %ExHuobi.Config{
+  #   api_key: System.get_env("HUOBI_API_KEY"),
+  #   api_secret: System.get_env("HUOBI_API_SECRET")
+  # }
```
Comment (sirius248) [inline test/futures/rest/account_test.exs]: If this code unused, we can remove it

**Comment by sirius248** [comment]
@lamtrhieu I had a few small comments. Can you fix those. Then we're good to go

File: test/futures/rest/account_test.exs (hunk 5 above)
Comment (lamtrhieu) [reply test/futures/rest/account_test.exs]: @kimquy I have fixed all the comment. Please help to review again. 
Thanks

**Review by sirius248 — APPROVED** [review]

**Pull request merged**